modules/installation-osp-dpdk-binding-vfio-pci.adoc

```diff
@@ -13,145 +13,178 @@ Compute machines that connect to a virtual function I/O (VFIO) network require t
 $ openstack network show <VFIO_network_name> -f value -c id
 ----
 
-. Create a machine set on your cluster from the following template:
+. Create a Butane config file called `99-vhostuser-bind.bu` that binds the PCI device to the VFIO driver. As an example:
++
+[NOTE]
+====
+For more information about Butane, see 'Creating machine configs with Butane' in the 'Customizing nodes' section of the documentation.
```

Comment:
```suggestion
For more information about Butane, see "Creating machine configs with Butane" in the "Customizing nodes" section of the documentation.
```
Changing quote style. Single -> double outside of nested quotes context.

Thanks!